grizzly/common/status.py

```diff
@@ -683,6 +687,11 @@ def count(self, result_id, desc):
                     (timestamp, self._count[result_id], self.pid, result_id),
                 )
                 if cur.rowcount < 1:
+                    cur.execute(
+                        """SELECT pid FROM results WHERE result_id = ?;""",
+                        (result_id,),
+                    )
+                    unique = cur.fetchone() is None
```

Comment:
The name `unique` is not very descriptive. `initial` or `created` would be better.

Semantically, `unique` should still be true when there is one result_id match, not just when the first is created.